src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -29,29 +29,38 @@
     @FXML
     private HBox cardPane;
     @FXML
+    private Label id;
+    @FXML
     private Label name;
     @FXML
-    private Label id;
+    private Label gender;
     @FXML
     private Label phone;
     @FXML
+    private Label email;
+    @FXML
     private Label address;
     @FXML
-    private Label email;
+    private Label measurement;
+    @FXML
+    private Label remark;
     @FXML
     private FlowPane tags;
 
     /**
-     * Creates a {@code PersonCode} with the given {@code Person} and index to display.
+     * Creates a {@code PersonCard} with the given {@code Person} and index to display.
      */
     public PersonCard(Person person, int displayedIndex) {
         super(FXML);
         this.person = person;
         id.setText(displayedIndex + ". ");
         name.setText(person.getName().fullName);
+        gender.setText(" (" + person.getGender().value.symbol + ")");
```

Comment:
Is it better to write `... + person.getGender().getValueSymbol() ...` so that it's more intuitive?